Add `visual-question-answering` docs

jlondonob opened the pull request:
## Summary
Added `about.md` and `data.ts` to the `visual-question-answering` documentation. This PR addresses #362.

## Questions
I wrote the **Use Cases** section as "this model can be used to..." instead of "this model is used to..." since use cases for Visual Question Answering are still somewhat experimental. Would you like to use the later as most of the other tasks do?

merveenoyan commented on tasks/src/visual-question-answering/about.md:
```diff
@@ -0,0 +1,43 @@
+VQA models can be used to reduce visual barriers for visually impaired individuals by allowing them to get information about images from the web and the real world.
+
+### Unattended Surveillance
+Video Question Answering can be used to quickly extract valuable information from surveillance recordings. It can also be used to raise alarms in case it detects anomalous situations.
```
```suggestion
Video Question Answering can be used to quickly extract valuable information from surveillance recordings. 
```
I think for the last part different tasks are used.

jlondonob commented on tasks/src/visual-question-answering/data.ts:
```diff
@@ -0,0 +1,59 @@
+	],
+	models:        [
+		{
+			description: "Vision-and-Language Transformer (ViLT) model fine-tuned on VQAv2",
```
You are totally right!

jlondonob commented on tasks/src/visual-question-answering/data.ts:
```diff
@@ -0,0 +1,59 @@
+						score: 0.009,
+					},
+					{
+						label: "1",
```
The probabilities don't have to add up to 1 since they are not mutually exclusive for more complex questions, such as "what is on top of ...". 

Here's the output of the specific model: 
```python
>>> image =  Image.open("assets/elephant.jpeg")
>>> question = "Is there an elephant?"
>>> vqa_pipeline(image, question, top_k=3)
[{'score': 0.9998154044151306, 'answer': 'yes'}, 
 {'score': 0.009802112355828285, 'answer': 'no'}, 
 {'score': 0.0020384755916893482, 'answer': '1'}]
>>> 
```
Do you think it would make it more clear if we increase the number of outputs `top_k` to make the sum go way over 1 so this is not confusing?

I thought about removing the "1" label too, but decided to keep it because it kind of showed that VQA's outputs are not just binary. Maybe we could show this better by asking a different question, such as "where is the elephant?".

merveenoyan replied:
@jlondonob sure, you can also include a better input not asking for a yes/no question but something more like "What's in this image?" Thanks a lot for the explanation!

jlondonob commented on tasks/src/visual-question-answering/about.md:
```diff
@@ -0,0 +1,43 @@
+
+## Inference
+
+You can infer with Visual Question Answering models using the `vqa` (or `visual-question-answering`) pipeline. You need to install PIL first  (`pip install pillow`).
```
Sure thing!

jlondonob commented on tasks/src/visual-question-answering/data.ts:
```diff
@@ -0,0 +1,59 @@
+			},
+		],
+	},
+	isPlaceholder: true,
```
Perfect!

jlondonob commented on tasks/src/visual-question-answering/data.ts:
```diff
@@ -0,0 +1,59 @@
+	metrics:       [
+		{
+			description: "For open-ended and multiple-choice VQA.",
+			id:          "Accuracy",
```
Understood!

merveenoyan commented on tasks/src/visual-question-answering/about.md:
```diff
@@ -0,0 +1,43 @@
+## Use Cases
+
+### Automatic Medical Diagnostic
+Visual Question Answering (VQA) models can be used to help clinical staff diagnose conditions. For example, a doctor could input an MRI scan to the model and ask "what does this MRI scan show?" and get the answer to their question. 
```
We discussed with @meg-huggingface about this (given she studied ethical side of VQA task) it's better if we don't include this (the model can pick up spurious cues and might answer questions wrongly in life critical situations)

merveenoyan commented on tasks/src/visual-question-answering/about.md:
```diff
@@ -0,0 +1,43 @@
+VQA models can be used to reduce visual barriers for visually impaired individuals by allowing them to get information about images from the web and the real world.
+
+### Unattended Surveillance
+Video Question Answering can be used to quickly extract valuable information from surveillance recordings. 
```
For this one, I think a basic activity detection model would be enough and VQA is too overkill.

merveenoyan commented on tasks/src/visual-question-answering/about.md:
````diff
@@ -0,0 +1,43 @@
+#[{'score': 0.9998154044151306, 'answer': 'yes'}]
+```
+
+## Useful Resources
````
Can you copy and paste the explanation from the placeholder page for useful resources part? so others can keep contributing 🤗

merveenoyan reviewed (approved):
I only left two comments, will be happy to merge once we discuss & address it 🤗 thanks a lot for the great work!

merveenoyan commented:
@jlondonob do you need help on this? 🤗

jlondonob commented:
@merveenoyan sorry for the delay, I'm on vacation and forgot to complete these changes. I'll get them done today.

merveenoyan commented:
@jlondonob ah didn't want to make you feel in a rush or anything, sometimes contributors want help but are too shy to ask for it so I thought I'd ask you if you need one 😅

jlondonob commented:
@merveenoyan Thanks for your constant help 🚀! I removed the `Automatic Medical Diagnostic` and `Unattended Surveillance` sections, as well as included the placeholder for `Useful Resources`. I also updated the **example** to use the question you suggested.

Do you think the PR is ready to be merged?

merveenoyan commented on tasks/src/visual-question-answering/data.ts:
```diff
@@ -0,0 +1,64 @@
+		],
+	},
+	isPlaceholder: false,
+	metrics: [
```
```suggestion
	metrics:       [
```

merveenoyan reviewed (approved):
Let's get this merged!

merveenoyan commented:
@jlondonob what is your HF user name? I will add your name at the end to give you credits before we deploy it 🤗 congratulations on the merger!!! ✨

jlondonob commented:
@merveenoyan thanks a lot! My HF username is `jlondonobo`
